app/templates/components/preprint-form-header.hbs

```diff
@@ -26,10 +26,10 @@
 {{#if (and (not isOpen) (eq name 'Upload'))}}
     <div class="m-t-md preprint-header-preview">
         {{#if preprintNode}}
-            <p><em class="m-r-md"> {{t "components.preprint-form-header.location"}}: </em> {{get-ancestor-descriptor preprintNode}} {{preprintNode.title}}</p>
+            <p><em class="m-r-md"> {{t "components.preprint-form-header.location"}}: </em> {{get-ancestor-descriptor preprintNode}} {{safe-markup preprintNode.title}}</p>
```

Comment:
If the value ever comes back from the server with HTML in it*- wouldn't this cause the node title to be displayed as raw html, rather than displayed as written?

The `safe-markup` helper should be used in highly restricted, very limited cases where there is absolutely no user-entered input involved. I would tend to discourage its use for displaying values in this way. If you have any questions I'd be happy to show examples or discuss the idea; just stop by anytime.

Two sample cases:
1. If we remove bleach (an HTML sanitizer) from the backend in the future
2. If we get a filename from a source like google drive, where the rules for a filename are outside our control and we do not run any transformations/ sanitizing on these values